templates/add-business.html

```diff
@@ -0,0 +1,83 @@
+{% extends "base.html" %}
+{% block content %}
+  <form class="col s12" method="POST" action="{{ url_for('add_business') }}">
```

Comment:
All of the classes in this template seem to be from Materialize. Can they be changed to reflect Bootstrap 5 please? 😀 We are using Bootstrap 5 on this project. Once the classes are updated to reflect, we can merge.